### PR TITLE
GLT-3004: updated setup instructions for v4 scanner software

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

**Review by callunity — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 A Java client for accessing a Thermo Scientific VisionMate scanner via TCP/IP.
 
+## Compatibility
+
+* Scanner model AB-1850, software v3.2.1.10: works as is
+* Scanner model 312800, software v4.0.4.1: works with patches. There are bugs in this version of the VisionMate
+  software. Thermo Fisher has provided these two patches, which are required in order for this library to communicate
+  with the scanner software. Unfortunately, the patches have not yet been included in an official release. Installation
+  instructions are included in the archives.
+  * [TCPUpdate_dependancies.zip](patches/TCPUpdate_dependancies.zip)
```
Comment (avarsava) [inline README.md]: dependancies -> dependencies

File: README.md [hunk 2]
```diff
@@ -33,15 +45,37 @@ The unit tests make use of the mock server, so that a real server is not require
...
+   text character\), which can be chosen by entering '\[3\]'
+
+It is recommended to leave other settings at their default, but if you'd like to change anything else, ensure that the
+option is supported in the client (see below).
 
-It is recommended to leave other settings at their default, but if you'd like to change anything else, ensure that the option is supported in the client (see below).
+**Server Setup - V4 VisionMate Software**
+
+1. Enable the TCP/IP server: Export Options (yellow icon) > TCP Server \(a checkmark should appear to indicate it is
+   enabled\)
```
Comment (avarsava) [inline README.md]: Could we embed screenshots of these icons if they're unclear enough to need description? (Rack Formats too)

File: README.md [hunk 3]
```diff
@@ -33,15 +45,37 @@ The unit tests make use of the mock server, so that a real server is not require
...
+2. Set a suffic character: Export Options > TCP Server > Config > Data Field Suffix. The suggested character is ASCII
```
Comment (avarsava) [inline README.md]: suffic -> suffix

**Comment by djcooke** [comment]
@avarsava: all fixed

**Review by avarsava — APPROVED** [review]

**Pull request merged**